### PR TITLE
Make the path to ceylon and the module name configurable.

### PR DESCRIPTION
**Pull request by bjansen**

### CODE REVIEW

File: android/src/main/groovy/com/redhat/ceylon/gradle/android/CeylonAndroidExtension.groovy [hunk 1]
```diff
@@ -43,6 +43,16 @@ class CeylonAndroidExtension {
    */
   boolean skipJavaC
 
+  /**
+   * Absolute path to the ceylon executable.
+   */
+  def ceylonExecutable = "/usr/bin/ceylon"
```
Comment (FroMage) [inline android/src/main/groovy/com/redhat/ceylon/gradle/android/CeylonAndroidExtension.groovy:49]: Cool! I meant to do this indeed. Can we have it default to `$CEYLON_HOME/bin/ceylon` IFF it's defined?

File: android/src/main/groovy/com/redhat/ceylon/gradle/android/CeylonAndroidExtension.groovy [hunk 2]
```diff
@@ -43,6 +43,16 @@ class CeylonAndroidExtension {
...
+
+  /**
+   * Name and version of the main Ceylon module.
+   */
+  def mainModule = "default/1.0"
```
Comment (FroMage) [inline android/src/main/groovy/com/redhat/ceylon/gradle/android/CeylonAndroidExtension.groovy:54]: Well, the default module has no version.

File: android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy [hunk 3]
```diff
@@ -339,8 +337,7 @@ Depends: ${dependencies}
     args.add(androidRepo.absolutePath)
     args.add("--out")
     args.add(outputRepo.absolutePath)
-    // FIXME: figure that one out
-    args.add("fr.epardaud.testjava.testjava/1")
+    args.add(project.androidCeylon.mainModule)
```
Comment (FroMage) [inline android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy:340]: Well, in theory this is all auto-detected from the modules defined in `source`. But I agree it's a better temporary step ;)

**Comment by FroMage** [comment]
Thanks.

File: android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy (hunk 3 above)
Comment (bjansen) [inline android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy:340]: What happens if there's more than one module in `source`? Do we have to jigsaw 'em all, or can the tool detect which module is the "main" one?

Comment (FroMage) [inline android/src/main/groovy/com/redhat/ceylon/gradle/android/MyCeylonCompileTask.groovy:340]: I have to extend that tool to allow for more than one module.

**Pull request merged**